lohica-js-network/src/components/inputComponents/AgeInput.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {FormGroup, ControlLabel, Col, FormControl} from 'react-bootstrap';
+import {connect} from 'react-redux';
+import { validateAge } from '../../actions/inputAction'
+
+class AgeInput extends React.Component {
+    constructor(props){
+        super(props);
+
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
+        if(e.target.value){
```

Comment:
thise if...else statement could be optimized. This is more like recomendation

Reply:
did optimization